templates/main.php

```diff
@@ -167,46 +167,46 @@ class="collection"
             <div class="app-navigation-entry-menu bubble sortorder-dropdown">
```

Comment:
Hm, I never considered this a bug. But I am ok with keeping the menu open after clicking the popover.